lib/monobank/auth/corporate.rb

```diff
@@ -0,0 +1,35 @@
+module Monobank
+  module Auth
+    class Corporate
+      def initialize(private_key:, key_id: nil, request_id: nil)
+        @private_key = init_key(private_key:)
+        @key_id = key_id
+        @request_id = request_id
+      end
+
+      def to_headers(pathname:)
+        time = Time.now.to_i
+
+        {
+          'X-Time' => time.to_s,
+          'X-Sign' => Base64.strict_encode64(sign(pathname:, time:))
+        }.tap do |headers|
+          headers['X-Key-Id'] = key_id unless key_id.nil?
+          headers['X-Request-Id'] = request_id unless request_id.nil?
```

Comment:
`/personal/auth/registration`, `/personal/auth/registration_status`, `/corp/webhook`, and `/corp/settings` don't have a request_id